features/utilities/src/main/feature/feature.xml

```diff
@@ -191,8 +193,39 @@
         <feature>jackson</feature>
         <bundle>mvn:com.github.luben/zstd-jni/${zstd-jni.version}</bundle>
         <bundle>wrap:mvn:net.sf.jwordnet/jwnl/${jwnl.version}</bundle>
-        <bundle>mvn:org.apache.tika/tika-core/${tika.version}</bundle>
-        <bundle>mvn:org.apache.tika/tika-bundle-standard/${tika.version}</bundle>
+        <bundle>mvn:ddf.thirdparty/tika/${project.version}</bundle>
```

Comment:
unsure why jackson feature included tika bundle. not important, just odd.

Reply:
I misread, this is actually the tika feature, it just includes jackson.